timesketch/api/v1/resources_test.py

```diff
@@ -205,6 +248,53 @@ def test_attempt_to_delete_protected_sketch(self):
         response = self.client.delete("/api/v1/sketches/4/")
         self.assert403(response)
 
+    def test_attempt_to_delete_archived_sketch(self):
+        """Authenticated request to archive a sketch."""
+        self.login()
+
+        # Create sketch to test with
+        data = dict(
+            name="test_delete_archive_sketch",
+            description="test_delete_archive_sketch",
+        )
+        response = self.client.post(
+            "/api/v1/sketches/",
+            data=json.dumps(data, ensure_ascii=False),
+            content_type="application/json",
+        )
+        created_id = response.json["objects"][0]["id"]
```

Comment:
We should consider outsourcing the creation of test sketches into a helper function. Just as a general idea for the future, not blocking on your PR.